setup.py

```diff
@@ -184,7 +184,7 @@ def run(self):
         'pytest'
     ],
     'install_requires': [
-        'ipywidgets>=7',
+        'ipywidgets==7',
```

Comment:
I think this should be both `>=7` and `<8`

https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#install-requires

Reply:
Yikes, what a weird syntax.  Thanks for pointing this out!